tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature

```diff
@@ -2300,3 +2301,15 @@ Feature: Create a share link for a resource
       """
     And the public should be able to download file "textfile.txt" from the last link share with password "%public%" and the content should be "to share"
     And the public download of file "textfile.txt" from the last link share with password "$heLlo*1234*" should fail with HTTP status code "401" using shareNg
+
+
+  Scenario: remove link share of a project drive
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has created the following space link share:
+      | space           | projectSpace |
+      | permissionsRole | view         |
+      | password        | $heLlo*1234* |
+    When user "Alice" delets the link share of space "projectSpace"
```

Comment:
if we are dealing with project space then it is good to specify with endpiont we are using. And we can search/think if we need to have tests other another endpoint
```suggestion
    When user "Alice" deletes the last link share of space "projectSpace" using root endpoint of the Graph API
```